README.md

```diff
@@ -78,6 +92,37 @@ For information about contributing to this project please refer to this
 * Releasing new versions
 * Docker development
 
+## CI/Testing Strategy
+
+The following parameters are considered in the testing strategy
+in the CI for this project:
+### Database
+
+We are testing this starter kit against PostgreSQL and MySQL.
+### Ruby Version
+
+We are testing this starter kit against the latest Ruby version
+and the last minor of the previous major. Eg: 3.2 and 2.7.
+
+### Rails version
+
+We are testing this starter kit against the last Rails version
+at the time the corresponding Solidus version has been released.
+Eg. Solidus v3.3 tests against Rails 7.0
+
+### Solidus Version
+
+- `main` branch will test installing itself over Solidus master only.
+- each `vX.X` branch will only test installing itself over the
+  corresponding Solidus version.
```

Comment:
I guess we only need to test the current `vX.X` branch, for older ones it doesn't make sense.